jenkins/test-image/Dockerfile

```diff
@@ -43,6 +43,13 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get install -y \
 	--no-install-recommends \
 	&& rm -rf /var/lib/apt/lists/*
 
+# Download and symlink etcd. We need this for verification and integration tests.
+RUN export ETCD_VERSION=v3.0.4; \
```

Comment:
This is super cool! I love that changing the ETCD version requires a) a PR to update this line and b) pushing a new image.

On the other hand, is there any chance this can more closely follow the instructions at https://github.com/kubernetes/kubernetes/blob/master/docs/devel/testing.md#install-etcd-dependency? It is not clear that hack/install-etcd.sh does the following... in fact this says it installs it into `./third_party/etcd`.